Security/src/CVE-2023-23397/CVE-2023-23397.ps1

```diff
@@ -724,14 +724,16 @@ begin {
             }
         }
 
+        # Specify secret expiration time which must be in ISO 8601 format and is always in UTC time
+        $pwdEndDateTime = ((Get-Date).AddDays(7)).ToString("yyyy-MM-ddTHH:mm:ssZ")
```

Comment:
This isn't UTC time.

![image](https://user-images.githubusercontent.com/22776718/233655734-607b46b5-abca-480b-986e-36c9d8310731.png)

Reply:
Also, why did we change it to be a variable if we only use it in the 1 location?

Reply:
Hm wondering if this could be the reason for the HTTP 400 errors that we see. Good catch indeed!

Reply:
The correct way to do this is:

```powershell
[DateTime]::UtcNow.ToString("o")
```

Reply:
([DateTime]::UtcNow).AddDays(7).ToString("o")

Reply:
Works as expected. Will make the change as suggested.

Reply:
Or if you prefer (Get-Date).ToUniversalTime().ToString("o"). Get-Date also returns a DateTime struct.

Reply:
Just saying we shouldn't be trying to manually produce ISO8601 format when DateTime already does that.

Reply:
Yes, I wasn't aware that we had something ready. `([DateTime]::UtcNow).AddDays(7).ToString("o")` is fine.